test/integration-test.sh

```diff
@@ -5,25 +5,23 @@
 set -e
 
 readonly LOCAL_INSTALL_URL="file:///src/gruntwork-install"
+readonly SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 
 echo "Using local copy of bootstrap installer to install local copy of gruntwork-install"
 ./src/bootstrap-gruntwork-installer.sh --download-url "$LOCAL_INSTALL_URL" --version "ignored-for-local-install"
 
-echo "Using gruntwork-install to install a few modules from script-modules"
-gruntwork-install --module-name "vault-ssh-helper" --repo "https://github.com/gruntwork-io/script-modules" --tag "~>0.0.21"
-
-echo "Checking that the vault-ssh-helper installed correctly"
-/etc/user-data/vault-ssh-helper/download-ca-cert.sh --help
-
 echo "Using gruntwork-install to install a module from the module-ecs repo"
 gruntwork-install --module-name "ecs-scripts" --repo "https://github.com/gruntwork-io/module-ecs" --branch "v0.0.1"
 
 echo "Using gruntwork-install to install a module from the module-ecs repo with --download-dir option"
-gruntwork-install --module-name "ecs-scripts" --repo "https://github.com/gruntwork-io/module-ecs" --branch "v0.0.1" --download-dir "~/tmp"
+gruntwork-install --module-name "ecs-scripts" --repo "https://github.com/gruntwork-io/module-ecs" --branch "v0.0.1" --download-dir ~/tmp
```

Comment:
I don't know how this test ever passed. The `~` was not being expanded, so the test failed...